docs/appendices/release-notes/5.5.3.rst

```diff
@@ -97,3 +97,8 @@ Fixes
 
 - Fixed a performance regression introduced in 5.5.0 for aggregations on columns
   with the column store disabled.
+
+- Fixed an issue that caused replication failure leading to an unstable cluster
+  when ``INSERT... ON CONFLICT... UPDATE SET`` was run on a table with
+  non-deterministic column and one of table's column wasn't part of the
+  ``INSERT`` targets.
```

Comment:
Maybe worth to add an example?

Reply:
Added and also mentioned that it's a regression from 5.3.0